src/elements/content-sidebar/activity-feed/task-new/AssigneeAvatar.js

```diff
@@ -29,17 +29,21 @@ const StatusIcon = ({ status, ...rest }: { status: TaskCollabStatus }) => {
     }
 };
 
-const AssignmentStatus = React.memo<Props>(({ user, status, getAvatarUrl, className, ...rest }: Props) => (
-    <div className={classNames('bcs-task-assignment-status', className)} data-testid="task-assignment-status" {...rest}>
-        <Avatar className="bcs-task-assignment-avatar" user={user} getAvatarUrl={getAvatarUrl} />
+const AssigneeAvatar = React.memo<Props>(({ user, status, getAvatarUrl, className, ...rest }: Props) => (
+    <div
+        className={classNames('bcs-avatar-group-avatar-container', className)}
+        data-testid="task-assignment-status"
+        {...rest}
+    >
+        <Avatar className="bcs-avatar-group-avatar" user={user} getAvatarUrl={getAvatarUrl} />
```

Comment:
It seems like including the concept of `group` in the base AssigneeAvatar component is a bit of a separation of concerns issue. Couldn't it be used outside a group?

Reply:
I think maybe the confusion was with the naming? Now it's called `bcs-AvatarGroup-avatar`. 

It could perhaps be exposed independently eventually if needed, maybe something like `AvatarWithStatus`? But I think that's a few steps out.